examples/src/test/java/jdocs/testkit/TestKitDocExample.java

```diff
@@ -0,0 +1,24 @@
+package jdocs.testkit;
+
+//#projection-testkit-import
+import akka.actor.testkit.typed.javadsl.ActorTestKit;
+import akka.projection.testkit.javadsl.ProjectionTestKit;
+
+//#projection-testkit-import
+
+public class TestKitDocExample {
+
+
+    public TestKitDocExample() {
+        //#projection-testkit
+        ActorTestKit actorTestKit = ActorTestKit.create("testing-projection");
```

Comment:
better to show this in a Junit test using TestKitJunitResource, see CassandraProjectionTest

Reply:
I was explicitly trying to avoid bringing in test framework integration to stay independent and show the basics. 

But fine, I don't have strong opinions on that.

Reply:
Sure, we can take that approach, and refer to Akka's testing documentation.